### PR TITLE
Feature/dev 2467 price derivation

### PR DESCRIPTION
**Pull request by lapatric**

### CODE REVIEW

File: packages/react/src/definitions/price-step.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface PriceStep {
+  source: string;
+  from: string;
+  to: string;
+  price: number;
+  timestamp: Date;
+  invert(): PriceStep;
+}
```
Comment (davidleomay) [inline packages/react/src/definitions/price-step.ts]: What is this invert? Where is it implemented?

Comment (lapatric) [reply packages/react/src/definitions/price-step.ts]: It's defined here but maybe it's not relevant for the frontend?

https://github.com/DFXswiss/api/blob/95c085894c12d7a8dacf0c803ba987229d6b3009/src/subdomains/supporting/pricing/domain/entities/price.ts#L89

Comment (davidleomay) [reply packages/react/src/definitions/price-step.ts]: Yes, it's not "sent" to the frontend, so I would remove it.

Comment (lapatric) [reply packages/react/src/definitions/price-step.ts]: Removed it 👍

**Review by davidleomay — APPROVED** [review]

**Pull request merged**